.github/workflows/codeql-analysis.yml

```diff
@@ -36,25 +36,20 @@ jobs:
         # a pull request then we can checkout the head.
         fetch-depth: 2
 
-    # If this run was triggered by a pull request event, then checkout
```

Comment:
How come this bit was removed>

Reply:
It was due to a warning stating that: git checkout HEAD^2 is no longer necessary. See here https://github.com/ONSdigital/eq-questionnaire-runner/actions/runs/3839459914